package.json

```diff
@@ -10,6 +10,9 @@
   "directories": {
     "test": "test"
   },
+  "engines": {
+    "node": ">=8"
```

Comment:
You probably want to match this up with your linter, which seems to be targeting ES2018?

Reply:
Ehhh maybe it doesn't matter, I was just concerned with us taking >= 8.0.0 as a guarantee of functionality when we have no tests or linters that prove it. Probably doesn't matter.

Reply:
okay, thanks for the help though @robcresswell ❤️